payroll-processor-client/src/app/shared/env.service.ts

```diff
@@ -1,16 +1,17 @@
 import { Injectable } from '@angular/core';
-import { Environment } from '../../environments/environment-types';
+
 import { environment } from '../../environments/environment';
+import { Environment } from '../../environments/environment-types';
 
 @Injectable({
-  providedIn: 'root'
+  providedIn: 'root',
 })
 export class EnvService {
   get env(): Environment {
     return environment;
   }
 
   get apiRootUrl(): string {
-    return `${this.env}/api`;
+    return `${this.env.domain}/api`;
```

Comment:
Nice catch

Reply:
yeah this took me a minute to figure out why my local url was `localhost:7071/[Object object]/api/....`. Nice to have a local environment set up though.